README.md

```diff
@@ -1,8 +1,6 @@
 # HYPERLEDGER URSA
 
-[![Ursa CI](https://github.com/hyperledger/ursa/workflows/Ursa%20CI/badge.svg)](https://github.com/hyperledger/ursa/actions?query=workflow%3A%22Ursa+CI%22)
-[![Build status](https://ci.appveyor.com/api/projects/status/1htmp82mdvmfjjap?svg=true)](https://ci.appveyor.com/project/hyperledger/ursa)
-[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/2447/badge)](https://bestpractices.coreinfrastructure.org/projects/2447)
+[Hyperledger Ursa was moved to EOL at the TOC meeting on 27 APR 2023.](https://toc.hyperledger.org/meeting-minutes/2023/2023-04-27-TOC-meeting-record.html)
```

Comment:
Request the following being added.  @appetrosyan — not sure if you want to add a reference to Iroha here.

```
The components of Ursa that are still in active use are moved to their respective projects where they will continue to be maintained, including:

- CL Signatures to AnonCreds
- BBS+ Signatures to Aries
- BLS Signatures to Indy

The published Ursa crates remain available for those projects already using them.
```

Reply:
updated